packages/backend/common/models/review.js

```diff
@@ -1,9 +1,19 @@
 const Joi = require('@hapi/joi')
+const {id} = require('./user') 
 
 const ReviewSchema = Joi.object({
-  id: Joi.string()
-    .alphanum()
-    .required()
+  id: id,
+
+  date: Joi.date().timestamp().required(),
+
+  user_id: Joi.string().required(),
+
+  detail: Joi.string().required(),
+
+  //rating in stars(int 0-5) 
+  rating: Joi.number().required(),
```

Comment:
Is the `rating` here refer to the rating that the user rate for a club (either 1, 2, 3, 4 or 5) or the average rating that the club has?

If it is the first cast:
it should be something like 
```
rating: Joi.number().integer().min(1).max(5).required(),
```

If it is the second case:
```
rating: Joi.number().min(1).max(5).required(),
```

Reply:
I was thinking the average rating, I did not want to do integer so the reviews can be more precise. So we can have 3.5 stars, etc..

Reply:
I see, so we are in the 2nd case. So can you send the min and max like in the snippet above?

Reply:
yes, pushing commit now